extension/keysocket-youtube.js

```diff
@@ -1,31 +1,23 @@
 function onKeyPress(key) {
-    if (key === NEXT) {
-        var nextButton = document
-            .querySelector('.yt-uix-button-icon-playlist-bar-next,.yt-uix-button-icon-watch-appbar-play-next');
-        simulateClick(nextButton);
-    } else if (key === PLAY) {
-        var video = document.querySelector('#movie_player');
-
-        // The extension has no access to this function when using the HTML-5 <video>-player
-        // but we can still access the controls on the <video>-element itself
-        if (!video.getPlayerState) {
-            video = document.querySelector('#movie_player video'); // HTML-5 <video>-player
-
-            if (video.paused) {
-                video.play();
-            } else {
-                video.pause();
-            }
-        } else {
+    var video = document.querySelector('#movie_player');
+    if (!video.getPlayerState) { // HTML5 Player
+        if (key === NEXT) {
+            var nextButton = document.querySelector('.ytp-button-next');
+            simulateClick(nextButton);
+        } else if (key === PLAY) {
+            var playPauseButton = document.querySelector('.ytp-button-pause') || document.querySelector('.ytp-button-play');
+            simulateClick(playPauseButton);
+        } else if (key === PREV) {
+            var backButton = document.querySelector('.ytp-button-prev');
+            simulateClick(backButton);
+        }
+    } else { // Flash Player
+        if (key === PLAY) {
```

Comment:
It would not do anything. The current code faces the same issue because even when using the flash player the play controls in the playlist are not available (at least by default, there might be a way to turn it on that i'm just not seeing)